app/javascript/controllers/prev_next_controller.js

```diff
@@ -4,6 +4,8 @@ export default class extends Controller {
   static outlets = [ "content-list" ]
 
   prevNextMedia(evt) {
-    this.contentListOutlet.showMedia(evt);
+    const index = evt.currentTarget.dataset.prevNextIndexParam
+    const thumbnail = this.contentListOutlet.element.querySelector(`[data-content-list-index-param="${index}"]`)
+    thumbnail.click()
```

Comment:
This is kind of hacky basically you are going into the sidebar and clicking the next item. Can you add a comment about why we are doing things this way instead of something less strange.

Reply:
We need to have the fileUri in the event details.  The content list on already has the fileUri as a data parameter.  So we either do this or we need to put a duplicate list of file uris on the page. I'm not sure how we want to structure that as a comment.

Reply:
Maybe something along the lines of, making this work with showMedia would require adding duplicate data parameters so instead of duplicating the information we trigger the event in one place. Something along those lines?

Reply:
"making this work" doesn't make sense, because it does work just as it is.

Reply:
"making this work the same way it used to work" doesn't seem like a helpful comment.

Reply:
@jcoyne I think we need a comment that makes sure when someone comes upon this in 1 year and is like why are we doing things this way we have an answer. I am worried someone is going to see this, think it is inefficient or not good and change it to something else that breaks things.